snowpack/src/plugins/plugin-esbuild.ts

```diff
@@ -29,12 +29,18 @@ export function esbuildPlugin(config: SnowpackConfig, {input}: {input: string[]}
     },
     async load({filePath}) {
       esbuildService = esbuildService || (await startService());
-      const contents = await fs.readFile(filePath, 'utf8');
+      let contents = await fs.readFile(filePath, 'utf8');
       const isPreact = checkIsPreact(filePath, contents);
```

Comment:
Should this now check the jsxInject fragment as well?

```
checkIsPreact(filePath, jsxInject  + '\n' + contents);
```

Otherwise, this wouldn't work with a preact injection.

Reply:
Good catch, I'm moving the the `jsxInject` up before `checkIsPreact`